eq-author/src/App/page/PropertiesPanel/GroupedAnswerProperties/index.test.js

```diff
@@ -459,7 +459,7 @@ describe("Grouped Answer Properties", () => {
 
       expect(screen.getByTestId("MaxCharacterTooSmall")).toBeVisible();
       expect(screen.getByTestId("MaxCharacterTooSmall").textContent).toBe(
-        characterErrors.CHAR_MUST_EXCEED_9
+        "Error:" + String.fromCharCode(160) + characterErrors.CHAR_MUST_EXCEED_9
```

Comment:
```suggestion
        {`Error: ${characterErrors.CHAR_MUST_EXCEED_9}`}
```

ES6 Syntax. Omitted the `String.fromCharCode(160)` as that returns a whitespace, which we can put into the string.

Reply:
These changes have been resolved as we have since discussed them. As the error uses non-breaking space (character code 160) instead of a standard space (character code 32), the tests see them as different characters and therefore do not pass.